src/lib/transformers/tokend.js

```diff
@@ -96,22 +70,26 @@ class TokendTransformer {
    * @return {Promise<Properties>} properties that had $tokend values transformed
    */
   transform(properties) {
+    const seenProperties = [];
     const promises = collectTransformables(properties, []).map((info) => {
       const keyPath = info.get('keyPath');
+      const propertyName = keyPath[0];
```

Comment:
Might be worth it to throw a length check and some logging around `keyPath`. The way we're using Propsd right now it _should_ only have 1 element but it'd be worth surfacing whether this is always the case.

Reply:
All set, add a check if the keyPath length is not equal to 1 and log if that is the case